### PR TITLE
Reduce odds of conflicts in parallel tests

### PR DESCRIPTION
**Pull request by filariow**
Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

File: make/acceptance-tests.mk [hunk 1]
```diff
@@ -28,10 +28,10 @@ test-acceptance: test-acceptance-setup ## Runs acceptance tests
 	$(PYTHON_VENV_DIR)/bin/behave --junit --junit-directory $(TEST_ACCEPTANCE_OUTPUT_DIR) --no-capture --no-capture-stderr $(TEST_ACCEPTANCE_TAGS_ARG) $(EXTRA_BEHAVE_ARGS) test/acceptance/features
 
 .PHONY: test-acceptance-x
-test-acceptance-x: test-acceptance-setup ## Runs acceptance tests for WIP tagged scenarios
+test-acceptance-x: test-acceptance-setup kustomize controller-gen opm envtest ## Runs acceptance tests for WIP tagged scenarios
```
Comment (sadlerap) [inline make/acceptance-tests.mk:31]: `envtest` shouldn't be needed, so long as we're not running unit tests as well.

Comment (filariow) [reply make/acceptance-tests.mk]: nice, updated

**Comment by baijum** [comment]
When I tried to run this, I am getting errors like this:
```
,---------,-
| COMMAND : kind create cluster --config /home/baiju/primaza/primaza/out/acceptance-tests/temp/tmp8joasqq3 && kubectl wait --for condition=Ready nodes --all --timeout=600s
'---------'-
ERROR MESSGE: b'Creating cluster "applicationagentdeployment-60-primaza-main" ...\n \xe2\x80\xa2 Ensuring node image (kindest/node:v1.24.0) \xf0\x9f\x96\xbc  ...\n \xe2\x9c\x93 Ensuring node image (kindest/node:v1.24.0) \xf0\x9f\x96\xbc\n \xe2\x80\xa2 Preparing nodes \xf0\x9f\x93\xa6   ...\n \xe2\x9c\x97 Preparing nodes \xf0\x9f\x93\xa6 \nERROR: failed to create cluster: could not find a log line that matches "Reached target .*Multi-User System.*|detected cgroup v1"\n'
ERROR CODE: 1
ERROR MESSGE: b'Creating cluster "applicationagentdeployment-25-primaza-main" ...\n \xe2\x80\xa2 Ensuring node image (kindest/node:v1.24.0) \xf0\x9f\x96\xbc  ...\n \xe2\x9c\x93 Ensuring node image (kindest/node:v1.24.0) \xf0\x9f\x96\xbc\n \xe2\x80\xa2 Preparing nodes \xf0\x9f\x93\xa6   ...\n \xe2\x9c\x97 Preparing nodes \xf0\x9f\x93\xa6 \nERROR: failed to create cluster: could not find a log line that matches "Reached target .*Multi-User System.*|detected cgroup v1"\n'
ERROR CODE: 1
ERROR MESSGE: b'Creating cluster "applicationagentdeployment-94-primaza-main" ...\n \xe2\x80\xa2 Ensuring node image (kindest/node:v1.24.0) \xf0\x9f\x96\xbc  ...\n \xe2\x9c\x93 Ensuring node image (kindest/node:v1.24.0) \xf0\x9f\x96\xbc\n \xe2\x80\xa2 Preparing nodes \xf0\x9f\x93\xa6   ...\n \xe2\x9c\x97 Preparing nodes \xf0\x9f\x93\xa6 \nERROR: failed to create cluster: could not find a log line that matches "Reached target .*Multi-User System.*|detected cgroup v1"\n'
ERROR CODE: 1

Total execution time:  6m 8.3s
Exit code: 1
make: *** [make/acceptance-tests.mk:33: test-acceptance-x] Error 1
```

**Review by dperaza4dustbit — APPROVED** [review]
/lgtm

**Pull request merged**